.woodpecker/docker.yml

```diff
@@ -74,7 +74,7 @@ pipeline:
       repo: woodpeckerci/woodpecker-server
       dockerfile: docker/Dockerfile.server.multiarch
       platforms: *platforms_server
-      tag: next
+      tag: [next, next-${CI_COMMIT_SHA:0:10}]
```

Comment:
```suggestion
      tags: next, next-${CI_COMMIT_SHA:0:10}
```

Reply:
actually similar syntax is used on line 130, 143, etc.
does this still need updating after adding quotes?

Reply:
no that also should do it ...